packages/protocol/contracts/team/airdrop/ERC20Airdrop.sol

```diff
@@ -42,20 +42,27 @@ contract ERC20Airdrop is MerkleClaimable {
         vault = _vault;
     }
 
-    function _claimWithData(bytes calldata data, bytes memory extraData) internal override {
-        (address user, uint256 amount) = abi.decode(data, (address, uint256));
+    function claimAndDelegate(
```

Comment:
I'm ok with this approach, but we need to override the claim function because we would like to maximize the delegation transaction.